source/docs/steeltoe-management.html.markdown

```diff
@@ -33,47 +33,51 @@ or
 
 # 1.0 Management Endpoints
 
-Below is a table of all of the currently available Steeltoe management endpoints. More details on each is provided in upcoming sections.
+The following table describes all of the currently available Steeltoe management endpoints:
 
 |ID|Description|
 |---|---|
 |**health**|Customizable endpoint that reports application health information|
-|**info**|Customizable endpoint that reports arbitrary application information (e.g. Git Build info, etc.)|
+|**info**|Customizable endpoint that reports arbitrary application information (such as Git Build info and other details)|
 |**loggers**|Allows remote access and modification of logging levels in a .NET application|
-|**trace**|Reports a configurable set of trace information (e.g. last 100 HTTP requests)|
+|**trace**|Reports a configurable set of trace information (such as the last 100 HTTP requests)|
 |**dump**|Generates and reports a snapshot of the applications threads (Windows only)|
 |**heapdump**|Generates and downloads a mini-dump of the application (Windows only)|
 |**cloudfoundry**|Enables management endpoint integration with Pivotal Cloud Foundry|
 
+More detail on each endpoint is provided in upcoming sections.
+
 Note that the Steeltoe Management endpoints themselves support the following .NET application types:
 
-* ASP.NET - MVC, WebForm, WebAPI, WCF
+* ASP.NET (MVC, WebForm, WebAPI, WCF)
 * ASP.NET Core
 * Console apps (.NET Framework and .NET Core)
 
-When you use the provided Steeltoe components to expose the endpoints using HTTP, then you will find those components support ASP.NET Core.
+When you use the provided Steeltoe components to expose the endpoints using HTTP, you can see that those components support ASP.NET Core.
```

Comment:
This one is still off... The idea is that Steeltoe supports exposing endpoints over HTTP for ASP.NET Core applications.
Maybe more like this?
> Steeltoe currently includes support for exposing the Management endpoints over HTTP with ASP.NET Core